### PR TITLE
Update fees.mdx

### PR DESCRIPTION
**Pull request by c0deNoodle**
added info for PCL pool fees

### CODE REVIEW

File: docs/learn/astro-tokenomics/fees.mdx [hunk 1]
```diff
@@ -20,6 +20,13 @@ For constant product pools, this fee is 0.3%, divided as follows:
 - A 0.2% fee (2/3rds of the total fee) goes to the LPs for the relevant pool — this remains denominated in the native pool tokens and acts as a payment by increasing the size of the pools (thus increasing the number of tokens that LPs can redeem when they remove liquidity); and
 - A 0.1% fee (1/3rd of the total fee) goes to the Astral Assembly — this is used to purchase ASTRO from ASTRO liquidity pools. The purchased ASTRO goes into the xASTRO staking pool per a ratio set by the Astral Assembly.
 
+## Passive Concentrated Liquidity (PCL) Pools
+
+For passive concentrated liquidity pools, this fee is variable and ranges from 0.26% to 0.45% (or 0.03% to 0.45% for some liquid staking pairs) divided as follows:
+
+- 2/3rds of the total fee goes to the LPs for the relevant pool — this remains denominated in the native pool tokens and acts as a payment by increasing the size of the pools (thus increasing the number of tokens that LPs can redeem when they remove liquidity); and
```
Comment (epanchee) [inline docs/learn/astro-tokenomics/fees.mdx:27]: Fee is split 25% to xASTRO stakers and 75% to LPs on Osmosis and 50:50 on other outposts

File: docs/learn/astro-tokenomics/fees.mdx [hunk 2]
```diff
@@ -20,6 +20,13 @@ For constant product pools, this fee is 0.3%, divided as follows:
 - A 0.2% fee (2/3rds of the total fee) goes to the LPs for the relevant pool — this remains denominated in the native pool tokens and acts as a payment by increasing the size of the pools (thus increasing the number of tokens that LPs can redeem when they remove liquidity); and
 - A 0.1% fee (1/3rd of the total fee) goes to the Astral Assembly — this is used to purchase ASTRO from ASTRO liquidity pools. The purchased ASTRO goes into the xASTRO staking pool per a ratio set by the Astral Assembly.
 
+## Passive Concentrated Liquidity (PCL) Pools
+
+For passive concentrated liquidity pools, this fee is variable and ranges from 0.26% to 0.45% (or 0.03% to 0.45% for some liquid staking pairs) divided as follows:
```
Comment (epanchee) [inline docs/learn/astro-tokenomics/fees.mdx:25]: This fee is not strictly set by protocol and depends on instantiate params. It is indeed recommended as you mentioned but we have different settings, for example, in LUNA-BTC/ETH/SOL/BNB pairs on Terra (0.05%-0.30%) and (0.05%-0.20%) on Osmosis.

**Review by epanchee — CHANGES_REQUESTED** [review]

**Comment by c0deNoodle** [comment]
Updated based on feedback from @epanchee. Thanks!

**Review by epanchee — APPROVED** [review]
LGTM

**Pull request merged**